impy/models/sibyll.py

```diff
@@ -210,3 +210,67 @@ def generate_event(self):
         self.lib.decsib()
         self.conv_hepevt()
         return 0  # SIBYLL never rejects
+
+class Sibyll21(SIBYLLRun):
+    def __init__(self, event_kinematics, seed="random", logfname=None):
+        from impy.definitions import interaction_model_by_tag as models_dict
+        interaction_model_def = models_dict["SIBYLL21"]       
+        super(SIBYLLRun, self).__init__(interaction_model_def)
+        self.init_generator(event_kinematics, seed, logfname)
+        
```

Comment:
If you are not appalled by a bit of "Python metaprogramming", you could generate all these classes in a single loop, by generating Python source code as a string, which is then passed to `exec`. If you find this appalling, I think this is fine too. It is a lot to type once and repetitive, but very clear once written and you don't have to change this often.

Reply:
I believe there is also a way to generate classes programmatically. That would still be "metaprogramming" but not via generating Python source code as a string.

Reply:
For an example of (admittedly ugly) metaprogramming, see https://github.com/HDembinski/numba-stats/blob/main/src/numba_stats/_util.py#L57

Reply:
Yes, I was thinking about metaclass creation, but, again, there will be no autocompletion in this case.

Reply:
What I was proposing creates the exact same result with autocompletion and all. But no need to change this, this code is repetitive but easy to understand.